### PR TITLE
Add blog post for FerretDB v1.8.0

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by ptrfarkas — APPROVED** [review]

File: website/blog/2023-08-14-ferretdb-new-release-v180.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## New features
+
+In the new release, we've added support for the `$group` stage `_id` expression.
+
+With this update, users can now use more complex document structures for the `_id` field in `$group` aggregations; you can now group data using an expression like `{"$group": {_id: {"v": "$v"}}}`.
+
+We've also added support for the `$expr` evaluation query operator.
+This operator allows you to use aggregation expressions within the `$match` aggregation stage and the `find` command.
+For example, you can now execute queries like `db.values.find( { $expr: { $gt: [ "$v" , 2] } } )`.
+For the `$match` aggregation stage, you can use it like this: `{ $match: { $expr: { <aggregation expression> } } }`.
```
Comment (AlekSi) [inline website/blog/2023-08-14-ferretdb-new-release-v180.md:37]: Or could just use `{ <aggregation expression> }`, without both `$match` and `$expr`

File: website/blog/2023-08-14-ferretdb-new-release-v180.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## New features
+
+In the new release, we've added support for the `$group` stage `_id` expression.
+
+With this update, users can now use more complex document structures for the `_id` field in `$group` aggregations; you can now group data using an expression like `{"$group": {_id: {"v": "$v"}}}`.
+
+We've also added support for the `$expr` evaluation query operator, which allows you to use expressions within the `$match` aggregation stage, and the `find` command.
+For example, you can now execute queries like `db.values.find( { $expr: <expression> } )`.
+For the `$match` aggregation stage, you can use it like this: `db.col.aggregate({<aggregation expression>})`.
```
Comment (AlekSi) [inline website/blog/2023-08-14-ferretdb-new-release-v180.md:36]: There is no `$match` stage in the example that supposedly demonstrates the usage of it.

Please do your own proofreading and review first.

File: website/blog/2023-08-14-ferretdb-new-release-v180.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+slug: ferretdb-new-release-v180
+title: FerretDB v1.8.0 – New release
+authors: [alex]
+description: >
+  FerretDB v1.8.0 has just been released with new features, bug fixes, and improvements to the SQLite and PostgreSQL backend.
```
Comment (AlekSi) [inline website/blog/2023-08-14-ferretdb-new-release-v180.md:6]:
```suggestion
  FerretDB v1.8.0 has just been released with new features, bug fixes, and improvements to the SQLite and PostgreSQL backends.
```

File: website/blog/2023-08-14-ferretdb-new-release-v180.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Without further ado, let's check out some of the notable features in this release!
+
+## New features
+
+In the new release, we've added support for the `$group` stage `_id` expression.
+
+With this update, users can now use more complex document structures for the `_id` field in `$group` aggregations; you can group data using an expression like `{"$group": {_id: {"v": "$v"}}}`.
+
+We've also added support for the `$expr` operator in evaluation queries when using the `find` command, and in aggregation operations within the `$match` aggregation stage.
```
Comment (AlekSi) [inline website/blog/2023-08-14-ferretdb-new-release-v180.md:34]: Could you please explain to me what is "evaluation queries"? and how `$match` aggregation stage uses aggregation operations?

File: website/blog/2023-08-14-ferretdb-new-release-v180.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+For example, you can now execute queries like `db.values.find({ $expr: <expression> })`.
+And for the `$match` aggregation stage, you can use the operator like this: `db.col.aggregate({ $match: { $expr: <expression> } })`.
```
Comment (AlekSi) [inline website/blog/2023-08-14-ferretdb-new-release-v180.md:36]: Why would someone do something like that?

**Review by AlekSi — CHANGES_REQUESTED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**